### ISSUE TITLE
Add -h or --help option to tmuxlayout

### ISSUE THREAD

**Issue by adherzog** [jmaslak/Term-Tmux-Layout#4]
As a new user, I was a bit confused about how to use the `tmuxlayout` utility. My initial process was:

1) run `tmuxlayout`, expecting the help screen to come up. It appeared to be waiting for input (which I now see is by design), so I Ctrl-C out.

2) run `tmuxlayout -h`. Nothing. (This was when I just had one pane open, so it didn't affect my layout at all.)

3) run `tmuxlayout --help`. Nothing.

That's when I jumped over to metacpan and read the perldoc. (Not sure why I didn't try `man` or `perldoc` on my own system. Just didn't occur to me.)

Would you accept a PR that added a '-h'/'--help' option to the `tmuxlayout` utility? I'm thinking it would be a fairly short summary of usage, like most CLI tools, and it could point the user to the man page for details.

**Comment by jmaslak** [comment]
Yes, a -h/--help option would likely be very useful, and I'd be thrilled to accept that PR.
